src/components/SubPage/SubPage.css

```diff
@@ -12,43 +12,43 @@
         flex-wrap: nowrap;
       }
     }
+  }
 
-    &.has-image--left {
-      .SubPage__content {
-        flex-direction: row-reverse;
-      }
+  &--image-left-aligned {
+    .SubPage__content {
+      flex-direction: row-reverse;
+    }
 
-      .SubPage__content-image {
-        @media (min-width: 40rem) {
-          margin-right: 20px;
-        }
+    .SubPage__content-image {
+      @media (min-width: 40rem) {
+        margin-right: 20px;
       }
+    }
 
-      .SubPage__content-text {
-        text-align: left;
-        width: 100%;
+    .SubPage__content-text {
+      text-align: left;
+      width: 100%;
 
-        @media (min-width: 40rem) {
-          text-align: right;
-          width: 50%;
-        }
+      @media (min-width: 40rem) {
+        text-align: right;
+        width: 50%;
       }
     }
+  }
 
-    &.has-image--right {
-      .SubPage__content-image {
-        @media (min-width: 40rem) {
-          margin-left: 20px;
-        }
+  &--image-right-aligned {
+    .SubPage__content-image {
+      @media (min-width: 40rem) {
+        margin-left: 20px;
```

Comment:
DRY.

Simply add the rules that are duplicated in a single block of `&--image-left-aligned, &--image-right-aligned`, and then do the specific rules after.

```
.a, .b { thing1, thing2 }
.a { thing3 }
```
vs
```
.a { thing1, thing2, thing3 }
.b { thing1, thing2 }
```